python/vmaf/core/asset.py

```diff
@@ -466,6 +489,10 @@ def dis_str(self):
             w, h = self.dis_encode_width_height
             s += "_e_{w}x{h}".format(w=w, h=h)
 
+        if self.dis_encode_bitdepth:
```

Comment:
I would prefer `if self.dis_encode_bitdepth is not None`, because `if self.dis_encode_bitdepth` has ambiguity: it could be that `if self.dis_encode_bitdepth is not None` or `if self.dis_encode_bitdepth != 0`.